test/test_quantization.py

```diff
@@ -290,6 +290,72 @@ def checkQuantized(model):
         checkQuantized(model)
 
 
+class ScriptabilityTest(QuantizationTestCase):
+    def test_wrapper_traceable(self):
+        UnaryWrapper = nnq.modules.wrapper_module.UnaryWrapper
+        QUnaryWrapper = nnq.modules.wrapper_module.QuantizedUnaryWrapper
+        script_module = torch.jit.trace(UnaryWrapper(torch.cat),
+                                        (torch.tensor(0), torch.tensor(0)))
+        script_qmodule = torch.jit.trace(QUnaryWrapper(torch.cat),
```

Comment:
These tests will not work for tracing or scripting. This is because the operators can take dynamic length inputs, one can trace with a tuple of length two and then run with a tuple of different length. A test case that corresponds to actual use is:

An notebook with the outputs is at: https://our.internmc.facebook.com/intern/anp/view/?id=128232
```python
import torch.nn as nn

class UnaryWrapper(nn.Module):
    def __init__(self, operation):
        super(UnaryWrapper, self).__init__()
        self.operation = operation

    def forward(self, x):
        # type: (List[Tensor]) -> Tensor
        return self.operation(x)


class BinaryWrapper(nn.Module):
    def __init__(self, operation):
        super(BinaryWrapper, self).__init__()
        self.operation = operation

    def forward(self, x,y):
        return self.operation(x,y)


class testModel(nn.Module):
    def __init__(self):
        super(testModel, self).__init__()
        self.mycat = UnaryWrapper(torch.cat)
        self.myadd= BinaryWrapper(torch.add)
        self.mymul = BinaryWrapper(torch.mul)
    
    def forward(self,x):
        y = self.mycat([x ,x ,x])
        z = self.myadd(y, y)
        z = self.mymul(z,z)
        return z
  ```